lib/modules/manager/gradle/parser/handlers.ts

```diff
@@ -170,6 +169,10 @@ export function handleLongFormDep(ctx: Ctx): Ctx {
     return ctx;
   }
 
+  if (!regEx(/\d/).test(version)) {
```

Comment:
With 0fe94e7003311be2e393e213cfafa4b5b0e3734e, this check should be redundant here. The subsequent execution flow is: 

`parseDependencyString()` -> `isDependencyString()` -> `versionLikeSubstring()`